libasn1fix/asn1fix_cws.c

```diff
@@ -307,6 +307,8 @@ _asn1f_assign_cell_value(arg_t *arg, struct asn1p_ioc_row_s *row, struct asn1p_i
 			FATAL("Cannot find %s referenced by %s at line %d",
 				p, arg->expr->Identifier,
 				arg->expr->_lineno);
+			asn1p_ref_free(ref);
```

Comment:
This is a problem! There is no identifier "ref".

@brchiu Can you please help with this ASAP?

Thanks!